minfraud/webservice.py

```diff
@@ -26,17 +26,14 @@
 class Client:
     """Client for accessing the minFraud Score and Insights web services."""
```

Comment:
Factors too?

Reply:
I decided to delete "Score and Insights" as it also supports the report transactions endpoint and listing everything seemed a bit unwieldy.